### PR TITLE
[mega-styling]: Maige Dashboard

### PR DESCRIPTION
**Pull request by arihanv**
- Added shadcn
- Refactored code-base
- Added micro-interactions
- Made it look like vercel lol

### CODE REVIEW

**Comment by adamgobes** [comment]
🔥

File: tailwind.config.ts [hunk 1]
```diff
@@ -1,18 +0,0 @@
...
-			colors: {
-				panel: '#0A0A0A',
-				'panel-border': '#171717',
-				grey: '#555555'
-			}
-		}
-	}
-} satisfies Config
-
-export default tailwindConfig
```
Comment (tedspare) [inline tailwind.config.ts:18]: Was this removed intentionally?

**Review by tedspare — APPROVED** [review]
LGTM. You cooked

File: tailwind.config.ts (hunk 1 above)
Comment (arihanv) [reply tailwind.config.ts:18]: Yes, shadcn-ui automatically makes tailwind.config.js, so I just merged the content from both configs into one file -> tailwind.config.js

**Pull request merged**